inc/template-functions.php

```diff
@@ -411,6 +411,11 @@ function twenty_twenty_one_password_form( $post = 0 ) {
  * @return array
  */
 function twenty_twenty_one_get_attachment_image_attributes( $attr, $attachment, $size ) {
+
+	if ( 'custom-logo' === $attr['class'] ) {
```

Comment:
There may be other filters from plugins etc, adding more classes. I think it would be safer to check if `custom-logo` is _contained_ in the classes instead of it being the only class.

```suggestion
	if ( isset( $attr['class'] ) && false === strpos( $attr['class'], 'custom-logo' ) ) {
```